Makefile

```diff
@@ -223,6 +223,9 @@ deploy_hpp:
 validate-no-offensive-lang:
 	./hack/validate-no-offensive-lang.sh
 
+generate-metric-linter:
```

Comment:
why not call this `validate-metric-names`?

Reply:
Right. it sound better, fixing.

Reply:
@machadovilaca Note that we also check that the metric have help section and that the type is valid.

Reply:
do you think `lint-metrics` could be better maybe?